api/src/main/scala/Utils.scala

```diff
@@ -205,4 +207,22 @@ trait Utils {
       .query[SpatialKey, Tile, TileLayerMetadata[SpatialKey]](layerId)
       .where(Intersects(shape))
       .result
+
+  class MinWithoutNoData extends DoubleBinaryOperator {
+    override def applyAsDouble(left: Double, right: Double): Double =
+      if (isNoData(left))
+        if (isNoData(right)) Double.MaxValue
+        else right
+      else if (isNoData(right)) left
+      else math.min(left, right)
+  }
+
+  class MaxWithoutNoData extends DoubleBinaryOperator {
+    override def applyAsDouble(left: Double, right: Double): Double =
+      if (isNoData(left))
+        if (isNoData(right)) Double.MinValue
+        else right
+      else if (isNoData(right)) left
+      else math.max(left, right)
```

Comment:
May be more idiomatic to write these as a series of match/case statements, although that may also require using a `noData` constant rather than a function call. Think it might look something like....

```scala
(left, right) match = 
    case (noData, noData) => Double.minValue
    case (noData, right) => right
    case (left, noData) => left
    case (left, right) => math.max(left, right)
```

Reply:
Fixed in a8bcacf1d7f0e9bea78c840394463b7043e39554.